packages/css/recipes/menu-sheet.d.ts

```
@@ -0,0 +1,21 @@
+declare interface MenuSheetVariant {
+  
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)